huntsman/utils/config.py

```diff
@@ -18,52 +19,64 @@ class ConfigServer():
     '''
     Class representing the config server.
     '''
-    def __init__(self, config_file=None, **kwargs):
+    def __init__(self, config_file=None, parse=True, **kwargs):
         '''
-        
+
         '''
+        self.parse = parse
+
         if config_file is None:
-            config_file = os.path.join(os.environ['HUNTSMAN_POCS'],
-                                       'conf_files', 'device_info.yaml')           
+            config_file = os.path.expandvars(os.path.join('$HUNTSMAN_POCS',
+                                            'conf_files', 'device_info.yaml'))
+
         #Read the config file
-        self.config_ = load_config(config_files=[config_file], **kwargs)
-        
+        self.config_ = load_config(config_files=[config_file], parse=self.parse,
```

Comment:
What's the story with the trailing underscore? Should it be a leading underscore instead, to make the config a pseudo-private member variable with access controlled via the property getter & setter?

Reply:
Yep - will put it at the front.